### PR TITLE
Add portuguese translation

### PR DESCRIPTION
**Pull request by lays147**
Note: It should be added to the docs that the file `frontend/app_flowy/packages/flowy_infra/lib/language.dart` also need to be updated, so the translation can appear on the app.

### CODE REVIEW

File: frontend/app_flowy/assets/translations/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      "deleteAll": "Deletar todos",
+      "pageHeader": {
+        "fileName": "Nome do arquivo",
+        "lastModified": "Última modificação",
+        "created": "Criado"
+      }
+    },
+    "deletePagePrompt": {
+      "text": "Está página está na lixeira",
+      "restore": "Restaurar a oágina",
```
Comment (carnei-ro) [inline frontend/app_flowy/assets/translations/pt-BR.json:62]: página

**Comment by annieappflowy** [comment]
Thanks for your help! 

FYI: You can join our [Discord](https://github.com/AppFlowy-IO/appflowy/discussions) and follow the #translation-needed channel for the updates on UI strings that might need more translations going forward.

**Review by annieappflowy — APPROVED** [review]

File: frontend/app_flowy/assets/translations/pt-BR.json (hunk 1 above)
Comment (annieappflowy) [reply frontend/app_flowy/assets/translations/pt-BR.json:62]: Has this been resolved? @lays147 @carnei-ro

**Comment by MikeWallaceDev** [comment]
Thank you very much @lays147 for your translation!

I'm sorry it took long to merge in, that's my fault. I still hadn't developed the technic to merge in an older PR (so I couldn't merge it, and it just got older, and older...)

**Pull request merged**